app/javascript/src/patternflyStyles/loginPage.css

```diff
@@ -0,0 +1,6 @@
+@import '~@patternfly/patternfly/components/Login/login.css';
```

Comment:
This file contains the imports specific to components in the Login Pages (Login, request password, signup)